.gitignore

```diff
@@ -5,3 +5,4 @@ logs
 target
 __pycache__
 *.log
+wandb/
```

Comment:
add new line

Reply:
Fixed and added this setting in black formatted! 

Quick note, this end-of-file newline doesn't appear to be showing on GitHub, e.g. https://github.com/franklsf95/ray-data-eval/blob/main/.gitignore